internal/cluster/clustersetup/activity_install_ingress_controller.go

```diff
@@ -126,7 +143,7 @@ func (i *IngressControllerPostHook) Do(cluster CommonCluster) error {
 	}
 
 	// TODO: once we move this to an integrated service we must find a way to append tags to user configured annotations
-	if cluster.GetCloud() == pkgCluster.Amazon {
+	if input.Distribution == pkgCluster.EKS || input.Distribution == pkgCluster.PKE {
```

Comment:
**Required**: IMO this is not an equivalent replacement of the original condition. A PKE cluster theoretically could be on other cloud providers as well, such as Azure which is currently supported in Pipeline. I think we **MUST** keep the original condition here even if the propagation of the cloud provider information makes the code a bit more complex on the calling side.